Bio/NaiveBayes.py

```diff
@@ -62,10 +62,9 @@ def __init__(self):
 def calculate(nb, observation, scale=False):
     """Calculate the logarithmic conditional probability for each class.
 
-    - nb          - A NaiveBayes classifier that has been trained.
-    - observation - A list representing the observed data.
-    - scale       - Boolean to indicate whether the probability should be
-    scaled by ``P(observation)``.  By default, no scaling is done.
+        - nb          - A NaiveBayes classifier that has been trained.
+        - observation - A list representing the observed data.
+        - scale       - Boolean to indicate whether the probability should be scaled by ``P(observation)``.  By default, no scaling is done.
```

Comment:
Well that's one way to solve the line wrapping - but could you try to keep to 78 characters?